packages/dx-react-scheduler/docs/guides/appointments.md

```diff
@@ -0,0 +1,26 @@
+# React Scheduler - Appointments
+
+The Scheduler component provides few appointment types.
+
+## Related Plugins
+
+- [Appointments](../reference/appointments.md)
+- [AllDayPanel](../reference/all-day-panel.md)
+
+## One-Time Appointment
+
+The one-time appointment is specified start and end dates. These appointments are displayed in the following demo. To display one-time appointments in the views add the [Appointments](../reference/appointments.md) plugin.
```

Comment:
The one-time appointment is specified start and end dates
maybe better 
A one-time appointment is a regular appointment with specified start and end date?